index.html

```diff
@@ -144,6 +152,71 @@ <h1 class="entry-title">CODESHOWS</h1>
 
 
 
+      <div class="subcontent" id="projects">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+	  <div class="container-fluid">
+	    <button type="button" class="btn btn-info sidebarCollapse">
+	      <i class="fas fa-align-left"></i>
+	    </button>
+	  </div>
+	</nav>	
+	<div class="container projects">
+	  <div class="container1">
+	    <div class="main">
+	      <div class="heading"> PROJECTS ON CODESHOWS </div>
+	      <hr>
+	      <div id="info">
+		<p> This section contains the ongoing and completed projects by the club :</p> 
+	      </div>
+	      <div class="con">
+                <div class="contest1" onclick="showans('ans7')">
+                  <i class="fas fa-book"></i>
+                  PROJECT 1: Devlopment of Codeshows Website
+                  <i class="fas fa-sort-down" style="float:right"></i>
```

Comment:
Remove *Development of* and *Repository for* from the titles.
The final titles should be:
Codeshows Website
Interview-checklist
Telegram-bot
Codeshows-IDE

Reply:
And try to write a few lines for description. Its optional, if you can't we will do it in another PR